content/tutorials/spatial_crs_coding/index.Rmd

```diff
@@ -110,25 +128,27 @@ The R packages further down, and many that depend on them, now provide a means o
 
-In the below code chunks, this is demonstrated for several important geospatial R packages: **`sf`**, **`sp`** and **`raster`**.
+In the below code chunks, specifying a CRS from the EPSG database is demonstrated for several important geospatial R packages: **`sf`**, **`sp`** and **`raster`**.
 Other geospatial R packages should normally inherit their approach.
 
-First, some practical notes:
-
-- In order to remain compatible with GDAL 2 and PROJ 4, some packages will still internally derive a proj4string as well (notably by calling the core package `rgdal`).
-This happens even while you did not enter a proj4string.
-Note that the derived proj4string will not be used further if you're on GDAL 3 / PROJ ≥ 6, and a WKT2 string will be generated as well for actual use.
-In the presence of GDAL 3 / PROJ ≥ 6, you will (at the time of writing) get a **warning** about dropped keys in the generated proj4strings [^warnings], but in the meantime, for most geospatial R packages you can safely ignore this warning.
-- **Windows** users: to make sure that the `rgdal` and `sf` packages use GDAL 3 / PROJ ≥ 6 -- which is now highly advised: please **update** all your packages in a pure R console (not RStudio) that uses your latest installed R version, by running `update.packages(ask = FALSE, checkBuilt = TRUE)`.
+First, a practical note:
 
+- Some packages internally still derive a PROJ string as well.
+This happens even while you did not enter a PROJ string.
+Note that the derived PROJ string will not be used further if you're on GDAL 3 / PROJ ≥ 6, and a WKT2 string will be generated as well for actual use.
+In the presence of GDAL 3 / PROJ ≥ 6 and when using `sp` or `raster`, you will (at the time of writing) get a **warning** from `rgdal` about dropped keys in the generated PROJ strings [^warnings].
+You can safely ignore this warning on condition that you didn't define the CRS with a PROJ string.
+Also, you can suppress the warning with `options(rgdal_show_exportToProj4_warnings = "none")` in the beginning of your script (before loading `rgdal` or dependent packages).
 
-[^warnings]: The packages give a warning especially to make developers of _other_, dependent packages _aware_ that they should also make sure their functions do not _require_ proj4strings or use hardcoded proj4strings.
+[^warnings]: The packages give a warning especially to make developers of _other_, dependent packages _aware_ that they should also make sure their functions do not _require_ PROJ strings or use hardcoded PROJ strings.
 Instead, they should defer the handling of CRS representation to the basic geospatial R packages (i.e. dependent on the version of PROJ/GDAL).
 So, the appearance of these warnings marks a period of transition in order to let other packages become GDAL 3 and PROJ ≥ 6 compliant.
 And the good news is that most popular geospatial packages _have_ become GDAL 3 and PROJ ≥ 6 compliant!
```

Comment:
Not sure if you have provided a code snippet in the tutorial to see which versions of GDAL or PROJ are active. Loading the `sf` package will print a message with this information if I remember well.

Reply:
> Not sure if you have provided a code snippet in the tutorial to see which versions of GDAL or PROJ are active.

Yes, it's included, both for `sf` and `gdal`.